label_studio/core/all_urls.json

```diff
@@ -947,9 +941,9 @@
         "name": "ml:api:ml-train",
         "decorators": ""
     },
-      {
+    {
         "url": "/api/ml/<int:pk>/predict/test",
-        "module": "ml.api.MLBackendPredictAPI",
+        "module": "ml.api.MLBackendPredictTestAPI",
```

Comment:
why did this change??

Reply:
It was via auto generation, I will look into it